app/(gcforms)/[locale]/(user authentication)/auth/mfa/resend/components/client/ReVerify.tsx

```diff
@@ -25,20 +27,36 @@ export const ReVerify = (): ReactElement => {
   const [authErrorState, setAuthErrorState] = useState<Record<string, string | undefined>>({});
   const [resending, setResending] = useState(false);
 
-  // If there is no existing flow redirect to login
-  const { email, authenticationFlowToken }: { email?: string; authenticationFlowToken?: string } =
-    JSON.parse(sessionStorage.getItem("authFlowToken") || "{}");
-  if (!email || !authenticationFlowToken) {
-    router.push(`/${language}/auth/login`);
-  }
+  // Makes sure NextJS will not try to render sessionStorage on the server
+  const existingFlow = useRef<{ email?: string; authenticationFlowToken?: string; error?: string }>(
+    {}
+  );
+  useEffect(() => {
```

Comment:
I really wanted to use the new React hook `use()` but ran into some errors and an open bug. I wasn't able to find a work around so I sadly went with the old style.
For reference:
https://github.com/facebook/react/issues/26930
and then this
https://github.com/vercel/next.js/issues/51477